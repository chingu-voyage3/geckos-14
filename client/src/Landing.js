import React, { Component } from 'react';
import logo from './assets/logo.svg';
import './assets/App.css';

class Landing extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <h1 className="App-title">Welcome to DashThings.io</h1>
        </header>
        <div className="App-intro">
            BUILD, CONNECT AND SHARE.
        </div>
      </div>
    );
  }
}

export default Landing;
