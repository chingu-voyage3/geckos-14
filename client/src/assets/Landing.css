/*======================================
            General
======================================*/
.section {
  height: 100%;
  margin-top: 50px;
}


/*======================================
            Main Section
======================================*/
.welcome-text {
  margin-top: 15%;
  padding: 5%;
  line-height: 1.5em;
  letter-spacing: 0.3em;
}

.welcome-text-button {
  padding: 10px;
  width: 20%;
  margin: 40px auto;
}
/*======================================
            Section one
======================================*/
.features-one {
  margin-top: 70px;
  padding: 50px;
  letter-spacing: 0.1em;
}

/*======================================
            Section two
======================================*/
.features-two {
  margin-top: 170px;
  padding: 80px;
}

/*======================================
            Section three
======================================*/
.features-three {
  /* margin-top: 50px; */
  padding: 80px;
}
