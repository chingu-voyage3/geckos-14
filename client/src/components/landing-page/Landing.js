import React, {Component} from 'react';
import Main from './Main';
import SectionOne from './SectionOne';
import SectionTwo from './SectionTwo';
import SectionThree from './SectionThree';


class Landing extends Component {
  render() {
    return (
      // Returns the four sections that makeup the landing page
      <div className="container">
        <Main />
        <SectionOne />
        <SectionTwo />
        <SectionThree />
      </div>
    );
  }
}

export default Landing;
