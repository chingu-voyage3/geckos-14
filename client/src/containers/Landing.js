import React from 'react';
import '../assets/Landing.css';
import GettingStartedButton from '../components/landing/GettingStartedButton';
import WelcomeSection from '../components/landing/WelcomeSection';
import SectionOne from '../components/landing/SectionOne';
import SectionTwo from '../components/landing/SectionTwo';
import SectionThree from '../components/landing/SectionThree';


const Landing = () => {
  return (
      <div className="container">
        <WelcomeSection
          heading="Sync, Manage and Control your Devices"
          welcomeText="DashThings.io is dedicatd to DIYers in need of an easy way to build user-friendly Dashboards for their Web of Things Devices"/>
        <SectionOne />
        <SectionTwo />
        <SectionThree />
      </div>
  );
};

export default Landing;
