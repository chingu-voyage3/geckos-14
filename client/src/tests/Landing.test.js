import React from 'react';
import ReactDOM from 'react-dom';
import Landing from '../containers/Landing';

import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const wrapper = shallow(<Landing />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
