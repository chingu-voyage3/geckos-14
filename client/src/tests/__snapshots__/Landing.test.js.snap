// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <h1
      className="App-title"
    >
      Welcome to DashThings.io
    </h1>
  </header>
  <div
    className="App-intro"
  >
    BUILD, CONNECT AND SHARE.
  </div>
</div>
`;
